### PR TITLE
Splitting name pending_contact_name in harvest.models.Property

### PR DESCRIPTION
**Pull request by farhanmasud**
Partially addresses [#116](https://github.com/LesFruitsDefendus/saskatoon-ng/issues/116)

Continued from [#190](https://github.com/LesFruitsDefendus/saskatoon-ng/pull/190/commits)

### CODE REVIEW

File: saskatoon/fixtures/harvest-property.json [hunk 1]
```diff
@@ -51,7 +52,8 @@
     "is_active": true,
     "authorized": true,
     "pending": true,
-    "pending_contact_name": "Nouveau Proprio",
+    "pending_contact_first_name": "Nouveau",
+    "pending_contact_family_name": "Proprio",
```
Comment (2ynn) [inline saskatoon/fixtures/harvest-property.json:56]: that's good! Have you tried to load the fixture with `python saskatoon/manage.py loaddata saskatoon/fixtures/harvest-property.json` ?

Comment (2ynn) [reply saskatoon/fixtures/harvest-property.json:56]: please also test `saskatoon.json`

Comment (farhanmasud) [reply saskatoon/fixtures/harvest-property.json:56]: @2ynn I haven't tested `python saskatoon/manage.py loaddata saskatoon/fixtures/harvest-property.json` yet. I'm checking and letting you know about this

File: saskatoon/sitebase/templates/app/detail_views/property/breadcomb.html [hunk 2]
```diff
@@ -15,8 +15,8 @@
                                     <h1>
                                     {% if property.owner %}
                                         {{ property.title }}
-                                    {% elif property.pending_contact_name %}
-                                        {{ property.pending_contact_name }}
+                                    {% elif property.pending_contact_first_name or property.pending_contact_family_name %}
+                                        {{ property.pending_contact_first_name }} {{ property.pending_contact_family_name }}
```
[the remark this answers is not in the document]
Comment (farhanmasud) [reply saskatoon/sitebase/templates/app/detail_views/property/breadcomb.html:19]: @2ynn got it. Updating accordingly.

File: saskatoon/harvest/utils.py [hunk 3]
```diff
@@ -23,11 +17,12 @@ def get_similar_properties(pending_property):
...
-        if name:
-            pass
-            # query |= Q(owner__person__first_name__icontains=first_name)
-            # query |= Q(owner__person__last_name__icontains=last_name)
+        first_name = p.pending_contact_first_name
+        family_name = p.pending_contact_family_name
+        if first_name and family_name:
+            q_first_name = Q(owner__person__first_name__icontains=first_name)
+            q_family_name = Q(owner__person__family_name__icontains=family_name)
+            query |= (q_first_name & q_family_name)
```
[the remark this answers is not in the document]
Comment (farhanmasud) [reply saskatoon/harvest/utils.py:25]: @2ynn okay, when PRs for [#188](https://github.com/LesFruitsDefendus/saskatoon-ng/issues/188) are merged, I'll get back to this and will create a new PR for these changes

**Review by 2ynn — APPROVED** [review]
Thanks a log @farhanmasud, nice job with the migration files!

**Comment by farhanmasud** [comment]
@2ynn thanks for merging this. I wasn't able to get back to this for making the reviewed changes in the last couple of days, sorry about that. I'll go through your reviews and will create a PR for enhancements requested here. Thanks.

**Pull request merged**